app/models/registration.rb

```diff
@@ -2,5 +2,5 @@ class Registration < ApplicationRecord
   belongs_to :user
   belongs_to :course
 
-  validates :role, inclusion: { in: %w(STUDENT ASSISTANT) }
+  validates :role, inclusion: { in: %w(STUDENT ASSISTANT PROFESSOR) }
```

Comment:
That was from before "ASSISTANT" was clarified as being either professor or TA. Allow me to remove that real quick.